### ISSUE TITLE
Suggestion: Search History could also include channel name.

### ISSUE THREAD

**Issue by unoukujou** [futo-org/grayjay-android#1205]
### Use Cases

When searching the history, it doesn't search the channel's name.

If I watched a couple of videos from some certain channel but I don't remember what those videos were and I want to try to find them again, I can just search for the channel's name in my history and find those videos but it's impossible to do so right now as the search only looks at the video title.

**Comment by SdoggaMan** [comment]
Agree. I'd also like to see time or date watched in this menu, so I can search for, say, something I know I watched some time last week. 

FWIW (slightly unrelated) I'd like to see History support full options like add to queue and playlists, but that's more for another topic.
